submissions/Mifaresss/task_js_oop/app.js

```diff
@@ -0,0 +1,82 @@
+const enemyConfig = [
+   {y: 60, speedX: 400, },
+   {y: 145, speedX: 320, },
+   {y: 230, speedX: 270, },
+]
+class Enemies {
+   constructor(y, speedX) {
+      this.sprite = 'images/enemy-bug.png';
+      this.x = 0;
+      this.y = y;
+      this.speedX = speedX;
+   }
+   update(dt) {
+      this.x += this.speedX * dt;
+      if (this.x > 500) this.x = -100;
+      this.checkClash();
+   }
+   checkClash() {
+         if (player.x < this.x + 77 && player.x + 77 > this.x && player.y < this.y + 66 && player.y + 66 > this.y) {

```

Comment:
Requirements with regards to magic numbers are not met.

Reply:
Fixed